src/components/AddButton/AddButton.tsx

```diff
@@ -0,0 +1,15 @@
+import classes from './AddButton.module.scss'
+
+type AddButtonProps = {
```

Comment:
Please move type to a separate file in the same directory and import it from there.

Reply:
In a similar case(where I described the interface in a separate file), I received this comment:"it is not necessary to create this in a separate file if it's not too big and you only use it once."
In this case it's also not too big and I only use it once.
In general, I can move the type to a separate file, but I would to know why you want to do it this way?

Reply:
it is a matter of habit and a common code-style. if we agree to always put it in a separate file that would be the flow, but if there is no agreement I usually recommend following common sense. that being said, creating a file for only 1 variable that's gonna be used only once feels redundant usually. but then again, if we agree that this is our team's code style - it would be mandatory to put such types separately.